src/main/java/com/avispl/symphony/dal/avdevices/microphone/sennheiser/tcc2/dto/audio/voice/VoiceLiftDTO.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Copyright (c) 2023 AVI-SPL, Inc. All Rights Reserved.
+ */
+
+package com.avispl.symphony.dal.avdevices.microphone.sennheiser.tcc2.dto.audio.voice;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * VoiceLiftDTO class provides during the monitoring and controlling process
+ * VoiceLiftDTO class includes the active value
+ *
+ * @author Kevin / Symphony Dev Team<br>
+ * Created on 3/16/2023
+ * @since 1.0.0
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Is it used to transfer the Json data, if not please remove the @JsonIgnoreProperties(ignoreUnknown = true). Please check another Object also